lib/sample_uploader/utils/importer.py

```diff
@@ -48,16 +54,17 @@ def load_file(
     """"""
     if sample_file.endswith('.tsv'):
         # df = pd.read_csv(sample_file, sep="\t", parse_dates=date_columns, header=header_row_index)
-        df = pd.read_csv(sample_file, sep="\t", header=header_row_index)
+        df = pd.read_csv(sample_file, sep="\t", header=header_row_index, skip_blank_lines=False)
     elif sample_file.endswith('.csv'):
         # df = pd.read_csv(sample_file, parse_dates=date_columns, header=header_row_index)
-        df = pd.read_csv(sample_file, header=header_row_index)
+        df = pd.read_csv(sample_file, header=header_row_index, skip_blank_lines=False)
     elif sample_file.endswith('.xls') or sample_file.endswith('.xlsx'):
-        df = pd.read_excel(sample_file, header=header_row_index)
+        df = pd.read_excel(sample_file, header=header_row_index, skip_blank_lines=False)
     else:
         raise ValueError(f"File {os.path.basename(sample_file)} is not in "
                          f"an accepted file format, accepted file formats "
                          f"are '.xls' '.csv' '.tsv' or '.xlsx'")
+    df.dropna(how='all', inplace=True)
```

Comment:
This, combined with the change above it (skip_blank_lines=False) makes pandas number the dataframe rows to real CSV rows, so that the dataframe index is equal to the rows distance from header_row_position, rather than skipping numbers on blank lines (see error_file.csv)

Reply:
ah i see, still not sure if that means if there is a table like this:
name, col 1, col 2
n1.     ,  hi.  , hello 
n2.    ,         , hello

wouldnt it skip/remove the row with `name=n2.` ?

Reply:
no the `how='all',` makes it only remove entirely empty rows